public/app/core/services/toggleTheme.ts

```diff
@@ -8,14 +8,15 @@ import { contextSrv } from '../core';
 import { PreferencesService } from './PreferencesService';
 
 export async function toggleTheme(runtimeOnly: boolean) {
-  const currentTheme = config.theme;
+  const currentTheme = config.theme2;
   const newTheme = createTheme({
     colors: {
       mode: currentTheme.isDark ? 'light' : 'dark',
     },
   });
 
   appEvents.publish(new ThemeChangedEvent(newTheme));
+  config.theme2.isDark = newTheme.isDark;
```

Comment:
@torkelo gave it a shot. not sure about this change, but without it you can't toggle the theme back... 🤔 

i'm trying to avoid manually calling `location.reload()` (as the command palette/normal preference update does)